web/src/people.ts

```diff
@@ -1717,7 +1717,7 @@ export function get_custom_fields_by_type(
     return filteredProfileData;
 }
 
-export function is_my_user_id(user_id: number | string): boolean {
+export function is_my_user_id(user_id?: number | string): boolean {
```

Comment:
This also can be undefined, which is handled in the next line

Reply:
I can make this a prep commit

Reply:
There are no reports in Sentry for the `user_id is a string in my_user_id` message, so we can remove `string`. Ideally we should not need to accept `undefined` either. Are you sure that `if (!user_id) {` line isn’t checking for `0` (the ID we use for spectators)?

(This is a great example of we shouldn’t write ambiguous falsiness checks like this.)

Reply:
Oh great point, yeah I'm not sure about that. I'll remove string for now and remove the undefined check and typescript can tell us later if we need it.